ci/integration-tests-windows.groovy

```diff
@@ -63,13 +63,15 @@ pipeline {
             steps {
                 unstash "dcos-exe"
                 unstash "dcos-${os}"
-                withEnv(["DCOS_TEST_URL=${master_ip}", "OS=${os}"]) {
+                withEnv(["DCOS_TEST_URL=${master_ip}", "OS=${os}", "BUILD_NUMBER=${BUILD_NUMBER}"]) {
                     withCredentials(credentials) {
                         bat '''
                             bash -exc " \
                             export PYTHONIOENCODING=utf-8; \
                             export CLI_TEST_SSH_USER=centos; \
                             export CLI_TEST_MASTER_PROXY=1; \
+                            export DCOS_DIR=$PWD/$BUILD_NUMBER; \
+                            mkdir -p $DCOS_DIR; \
```

Comment:
dcos-cli does this on its own, doesn't it? at least on dcos-ui-CI we don't need to do this i believe.

Reply:
yes by default it creates `.dcos` under `$HOME` here our homedir was not cleaned so the previous build interfere with the new one. 
I manually tested it with Jenkins https://jenkins.mesosphere.com/service/jenkins/job/public-dcos-cluster-ops/job/mesosphere-dcos-cli/job/core/job/integration-tests-windows/1968/console